src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+| sampling_frequency    | OPTIONAL but REQUIRED if `SamplingFrequency` is to `n/a` in `_nirs.json` | [number][]          | Sampling frequency of the channel in Hz.                                                                                                                                                                                                                                    |
+| orientation_component | OPTIONAL but REQUIRED if type is ACCEL, GYRO or MAGN                     | [string][]          | Description of the orientation of the ACCEL, GYRO, MAGN type. Either x, y, or z.                                                                                                                                                                                            |
+
+The following columns SHOULD be present:
+
+| **Column name**            | **Requirement level** | **Data type** | **Definition**                                                                                                                                                                                                          |
+| -------------------------- | --------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| wavelength_actual          | OPTIONAL              | [number][]    | Measured wavelength of light in nm. `n/a` for channels that do not contain raw NIRS signals (acceleration). This field is equivalent to `measurementList.wavelengthActual` in the SNIRF specification.                  |
+| description                | OPTIONAL              | [string][]    | Free-form text description of the channel, or other information of interest.                                                                                                                                            |
+| wavelength_emission_actual | OPTIONAL              | [number][]    | Measured emission wavelength of light in nm. `n/a` for channels that do not contain raw NIRS signals (acceleration). This field is equivalent to `measurementList.wavelengthEmissionActual` in the SNIRF specification. |
+| short_channel              | OPTIONAL              | [boolean][]   | Is the channel designated as short. The total number of channels listed as short channels should be stored in `ShortChannelCount` in `*_fnirs.csv`.                                                                     |
+
+Restricted keyword list for the channel types.
```

Comment:
Do you want to make this line more of a heading? Rather than just a sentence at the start of a paragraph.